### PR TITLE
Accept esperanto options and UMD bundles

### LINKED ISSUE #3: Support umd
Esperanto supports UMD, but it needs a [name in the options argument](https://github.com/esperantojs/esperanto/wiki/Converting-a-single-module#options) for the global variable to be exported.

### PR DESCRIPTION
**Pull request by leobalter**
Fixes #3

### CODE REVIEW

**Comment by leobalter** [comment]
Ops, I came too late. I'll try to rebase it as this PR adds more stuff.

**Comment by leobalter** [comment]
The PR is already rebased

File: test/expected/umd [hunk 1]
```diff
@@ -4,6 +4,6 @@
 	factory(global.name)
 }(this, function (name) { 'use strict';
 
-	name['default']+'';
+	name+'';
```
Comment (leobalter) [inline test/expected/umd:7]: and now I saw some order of error in the tests: esperanto is not dealing with the UMD name. It is a necessary option but it's not being processed to the given value.

**Comment by leobalter** [comment]
I've updated with a new commit to expose the export method to improve tests.

**Comment by developit** [comment]
Thanks for this! Just released it in `0.3.0`.

**Pull request merged**